Library/Thaumatology/Thaumatology Magical Styles/Magical Style - The Onyx Path.gct

```diff
@@ -0,0 +1,10512 @@
+							"tags": [
+								"Magic Perk",
+								"Supernatural"
+							],
+							"base_points": 1,
+							"calc": {
+								"points": 0
+							}
+						},
+						{
+							"id": "3d439c5a-447d-4fcf-b5fc-5c015b193a99",
+							"type": "trait",
+							"name": "Shortcut to Power (@EitherSoulJarOrLich@)",
+							"reference": "TMS30",
+							"notes": "Waives spell prerequisites for chosen spell",
+							"tags": [
+								"Magic Perk",
+								"Supernatural"
+							],
+							"base_points": 1,
+							"calc": {
+								"points": 0
+							}
+						},
+						{
+							"id": "0a89bc1d-0af9-4415-9f30-3ec568b05e27",
+							"type": "trait",
+							"name": "Special Exercises (Undeath)",
+							"reference": "TMS31",
+							"notes": "Each level lets you buy one advantage on the Lich template",
+							"tags": [
+								"Magic Perk",
+								"Supernatural"
+							],
+							"levels": 1,
+							"points_per_level": 1,
+							"can_level": true,
+							"calc": {
+								"points": 0
+							}
+						},
+						{
+							"id": "2a35f84a-c046-49e0-b94f-5b58aba26cc9",
+							"type": "trait",
+							"name": "Special Exercises (Extra Life 1)",
+							"reference": "TMS31",
+							"tags": [
+								"Magic Perk",
+								"Supernatural"
+							],
+							"levels": 1,
+							"points_per_level": 1,
+							"can_level": true,
+							"calc": {
+								"points": 0
+							}
+						},
+						{
+							"id": "fd8d1a5e-8428-49cc-81cc-73568d6c18a5",
+							"type": "trait",
+							"name": "Adjustable Spell, Spell Enhancement (Affects Insubstantial)",
+							"reference": "TMS21",
+							"tags": [
+								"Magic Perk",
+								"Supernatural"
+							],
+							"base_points": 1,
+							"calc": {
+								"points": 0
+							}
+						},
+						{
+							"id": "92abe79c-1c3b-4481-85f2-cedc5038cd8d",
+							"type": "trait",
+							"name": "Spell Hardiness (@Damaging Spell@)",
+							"reference": "TMS31",
+							"notes": "DR1/level against spell chosen",
+							"tags": [
+								"Magic Perk",
+								"Supernatural"
+							],
+							"levels": 1,
+							"points_per_level": 1,
+							"can_level": true,
+							"calc": {
+								"points": 0
+							}
+						},
+						{
+							"id": "828ef7ed-b4f7-472f-996d-602fa0c6a538",
+							"type": "trait",
+							"name": "Spell Resistance (@Resisted Onyx Path Spell@)",
+							"reference": "TMS31",
+							"notes": "+3 to resistance rolls for selected spell",
+							"tags": [
+								"Magic Perk",
+								"Supernatural"
+							],
+							"base_points": 1,
+							"calc": {
+								"points": 0
+							}
+						}
+					],
+					"name": "Magical Style Perks: The Onyx Path",
+					"reference": "ThMS:12",
+					"notes": "You can buy 1 Style Perk per 10CP in the style's spells in addition to 1 General Magical Perk per 20CP in other spells.",
+					"disabled": true,
```

Comment:
Because this is constructed as a template, I thought it would be a good idea to only have the basic requirements active and keep the rest (Style Perk that you may only buy when all prereqs have been met; optional traits; etc.) either disabled (traits) or at 0CP invested (skills), so the Template could be applied as-is without flooding the character with things they might not (yet) have.

Reply:
Now that I see it: by that logic, I should also reduce all spells beyond Circle 1 to 0CP.

Reply:
As a template, you should just use the various choice mechanisms instead of disabling anything.